modules/kuberay-cluster/kuberay-autopilot-values.yaml

```diff
@@ -438,12 +256,12 @@ additionalWorkerGroups:
         - mountPath: /fluent-bit/etc/
           name: fluentbit-config
 
-  largeGroup:
+  gpuGroup:
```

Comment:
how does this work/ where is gpuGroup used?

Reply:
Ray Autoscaler scales these groups automatically based on demand. My current understanding is that it is able to detect requirements for each Ray job and choose the correct workers for the job.